prisma/migrations/20230620174150_/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- CreateEnum
+CREATE TYPE "StateType" AS ENUM ('NotStarted', 'InProgress', 'Completed', 'Failed', 'Canceled');
+
+-- AlterTable
+ALTER TABLE "State" ADD COLUMN     "type" "StateType";
```

Comment:
don't you want to add sql to fill the column based on current seed types?

Reply:
it would be possible to use non-null constraint after that

Reply:
it's impossible to do fill them here, because you need to write smth like that:
```
if (state.title === 'Draft') ...
```

I'm not sure it's nice. We'll fill state in prod db and them add new migration with non null constraint. https://github.com/taskany-inc/issues/issues/1128